packages/expo-calendar/src/Calendar.ts

```diff
@@ -143,7 +143,7 @@ export type RecurrenceRule = {
   endDate?: string;
   occurrence?: number;
 
-  daysOfTheWeek?: Array<{ dayOfTheWeek: DayOfTheWeek; weekNumber?: number }>;
+  daysOfTheWeek?: { dayOfTheWeek: DayOfTheWeek; weekNumber?: number }[];
```

Comment:
Looking at this I lean towards ["array-simple"](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/array-type.md#array-simple), what do you think?

Reply:
I lean toward strictly one or the other (or no rule) for linting in general if something is linted. Hybrid approaches often slow review as a reviewer unfamiliar with the lint may comment the opposite of what the lint rule enforces, causing thrash. I think the choice is either decide that both are fine in all cases (no rule) or having a hard (autofixed) enforcement one way or the other.

For this one in particular, prettier and linter autofix 100% so I lean toward the hard enforcement. In prior codebases I had enforced the generic variant of the rule so that's why most of my original code was written that way, but I've come to appreciate the succinctness of the array variant.

Reply:
That being said, I think you (and other contributors) are the most familiar with the culture surrounding the philosophy of this linter, so I'm open to any resolution here. My priority is speeding up code reviews and decreasing code review overhead at all costs.

Reply:
OK, let's keep the rule as-is. I think that decision will head off conversations explaining why we write `string[]` but not `(string | null)[]`.